2025/oos-executive-vote-2025-11-17-solana-bridge-migration.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Template - [Executive Vote] Out Of Schedule Spell - Solana Bridge Migration Spell 1 - November 17, 2025
+summary: Undertake the final steps of the Solana Bridge Migration.
+date: 2025-11-17T00:00:00.000Z
+address: "$spell_address"
+---
+
+# [Executive Proposal] Out Of Schedule Spell - Solana Bridge Migration Spell 1 - November 17, 2025
+
+The Core Facilitator, Sidestream, and Dewiz have placed an executive proposal into the voting system. SKY holders should vote for this proposal if they support the following alterations to the Sky Protocol.
+
+If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
+
+- The final steps of the Solana Bridge Migration will be executed.
```

Comment:
(I suggested "stage" before; suggesting it here again for consistency, but with less conviction. If it's changed, "actions" should be made singular)